lib/kafka_ex/protocol/create_topics.ex

```diff
@@ -138,7 +140,7 @@ defmodule KafkaEx.Protocol.CreateTopics do
   end
 
   @spec parse_response(binary, integer) :: [] | Response.t
-  def parse_response(message, api_version)
+  #def parse_response(message, api_version)
```

Comment:
Why is this commented out?

Reply:
I meant to actually remove that since it wasn't needed. Removing